.github/dependabot.yml

```diff
@@ -7,18 +7,18 @@ updates:
     schedule:
       interval: daily
 
-  - package-ecosystem: npm
-    directory: /backend
-    schedule:
-      interval: daily
-    commit-message:
-      prefix: "Backend"
-      include: "scope"
+  # - package-ecosystem: npm
```

Comment:
Isn't removing a better option? Even if it's just for testing purposes, we still have git to check the older values

Reply:
Sure!  Now it's removed.